cfgov/jinja2/v1/_includes/organisms/_vars-mega-menu.html

```diff
@@ -27,7 +27,7 @@
         'url':  '/about-us/the-bureau/',
     },
     'image': {
-        'src': '/static/img/fmc-about-us-540x300.jpg',
+        'src': 'img/fmc-about-us-540x300.jpg',
```

Comment:
It's handled in the macro: https://github.com/cfpb/cfgov-refresh/pull/2407/files#diff-f2ca90745a1b3ee1fd53aa8189a37e09R49